internal/config/config_test.go

```diff
@@ -140,6 +142,15 @@ func TestReadTCConfig(t *testing.T) {
 		assert.Equal(t, expected, config)
 	})
 
+	t.Run("Invalid IP address", func(t *testing.T) {
+		t.Setenv("TESTCONTAINERS_RYUK_ADDRESS", "invalid")
+		t.Cleanup(func() {
+			os.Unsetenv("TESTCONTAINERS_RYUK_ADDRESS")
+		})
```

Comment:
bug: this isn't needed [t.Setenv](https://pkg.go.dev/testing#T.Setenv) already adds a clean up function.
```suggestion
```